.isort.cfg

```diff
@@ -1,9 +1,9 @@
 # Licensed to the Apache Software Foundation (ASF) under one or more
-# contributor license agreements.  See the NOTICE file distributed with
+# contributor license agreements. See the NOTICE file distributed with
```

Comment:
I don't think we want to 'fix' the standard license header necessarily unless it's really hard to exclude. Is this failing PEP8?

Also does this now put a file at the project root? I spent time getting stuff out of the root and would prefer to keep everything under python/